app/Http/Requests/CreateAirlineRequest.php

```diff
@@ -20,4 +21,13 @@ public function rules(): array
             'cities' => ['array']
         ];
     }
+
+    public function toDto(): AirlineData
+    {
+        $name = $this->input('name');
+        $description = $this->input('description');
+        $cities = $this->input('cities');
+
+        return new AirlineData($name, $description, $cities);
```

Comment:
`$dto = new AirlineData();`
`$dto->name = $this->input('name');`

Reply:
`return new AirlineData(name: $name, description: $description, cities: $cities);`

Reply:
return new AirlineData(
 description: $description,
name: $name,
  cities: $cities
);